### ISSUE TITLE
pg.time.set_timer raises exception without init

### ISSUE THREAD

**Issue by illume** [pygame/pygame#2612]
Seems like a compatibility bug with pg.time.set_timer with pygame.init being called first.

Before this would work it seems?

From https://github.com/pygame/pygame/pull/2606

```
File "pygame\examples\mask.py", line 175, in main
    pg.time.set_timer(pg.USEREVENT, 33)
pygame.error: pygame is not initialized
```

**Comment by ankith26** [comment]
Yea, I introduced this regression back in 2.0.1, and I immediately fixed it in #2428
And also, I think there was another similar issue to this, so I think this issue is a duplicate

**Comment by ankith26** [comment]
Alright, I can confirm that #2428 fixes this issue, testing with the old pygame mask example, before building the PR, the mask example fails, but after that PR, the issue is fixed (with the old example)

So yea, this bug was my bad, but now it should be fixed
